lib/wasm/src/sections_translator.rs

```diff
@@ -17,6 +17,10 @@ use wasmparser;
 use std::str::from_utf8;
 use environ::ModuleEnvironment;
 
+// this is for no_std builds, but has no affect on regular builds
```

Comment:
One more consistency nit: in some places, these new `use` lines have a blank line between them and the other `use` lines, and in others they don't. Again, I don't have a strong feeling about which is better, but it'd be nice to be consistent.